app/Http/Controllers/Dormitory/Printing/PrintJobController.php

```diff
@@ -0,0 +1,185 @@
+<?php
+
+namespace App\Http\Controllers\Dormitory\Printing;
+
+use App\Enums\PrintJobStatus;
+use App\Http\Controllers\Controller;
+use App\Models\FreePages;
+use App\Models\PrintAccount;
+use App\Models\Printer;
+use App\Enums\PrinterCancelResult;
+use App\Models\PrinterHelper;
```

Comment:
`App\Models\PrinterHelper` does not exists, you probably meant `App\Utils\PrinterHelper`

Reply:
Good catch, namespace was wrong.